### PR TITLE
Improve logging output

### PR DESCRIPTION
**Pull request by timrogers**
This improves the tool's logging output, using the `winston` logging package to add the timestamp and colorized log level to logs:

Screenshot 2023-04-04 at 10 57 03

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -78,36 +79,36 @@ async function checkStatusAndArchiveDownload(migration: { org: string, migration
...
             return { migrationId: migration.migration_id, migrationStatus };
         } else {
             attempts++;
             if (attempts >= maxAttempts) {
-                console.log(`ERROR: Maximum number of attempts (${maxAttempts}) reached for migration ${migration.migration_id}.`);
+                logger.error(`Maximum number of attempts (${maxAttempts}) reached for migration ${migration.migration_id}.`);
                 return { migrationId: migration.migration_id, migrationStatus };
             } else {
-                console.log(`Waiting ${delayInMilliseconds / 1000} seconds before checking migration status again.`);
+                logger.error(`Waiting ${delayInMilliseconds / 1000} seconds before checking migration status again.`);
```
Comment (pmartindev) [inline src/main.ts]: Possibly change this to logger.warn vs logger.error?

Comment (pmartindev) [reply src/main.ts]: CleanShot 2023-04-04 at 11 50 44
vs
CleanShot 2023-04-04 at 11 51 25

**Comment by pmartindev** [comment]
Tested out the changes, and they all look good 👍

**Comment by timrogers** [comment]
@pmartindev Thanks!

**Review by pmartindev — APPROVED** [review]

**Pull request merged**